src/Momento.Sdk/Config/Transport/StaticTransportStrategy.cs

```diff
@@ -1,29 +1,44 @@
+using System;
 using System.Collections.Generic;
 
 namespace Momento.Sdk.Config.Transport;
 
 
 public class StaticGrpcConfiguration : IGrpcConfiguration
 {
-    public int NumChannels { get; }
+    public int NumChannels { get; set; }
 
-    public int MaxSessionMemory { get; }
-    public bool UseLocalSubChannelPool { get; }
-    public IDictionary<string, string> GrpcChannelConfig { get; }
+    public int MaxSessionMemory { get; set; }
+    public bool UseLocalSubChannelPool { get; set; }
+    private uint _deadlineMilliseconds;
+    public uint DeadlineMilliseconds
+    {
+        get => _deadlineMilliseconds;
+        set
+        {
+            if (value <= 0)
+            {
+                throw new ArgumentException($"Deadline must be strictly positive. Value was: {value}", "DeadlineMilliseconds");
+            }
+            _deadlineMilliseconds = value;
+        }
+    }
+    public IDictionary<string, string> GrpcChannelConfig { get; set; }
 
-    public StaticGrpcConfiguration(int numChannels, int maxSessionMemory, bool useLocalSubChannelPool)
+    public StaticGrpcConfiguration(int numChannels, int maxSessionMemory, bool useLocalSubChannelPool, uint deadlineMilliseconds = 5000)
```

Comment:
let's make this required and not provide a default

Reply:
Done